nodejs/lancedb/connection.ts

```diff
@@ -65,6 +66,8 @@ export interface CreateTableOptions {
    * The available options are described at https://lancedb.github.io/lancedb/guides/storage/
    */
   storageOptions?: Record<string, string>;
+  schema?: Schema;
+  embeddingFunction?: EmbeddingFunctionConfig;
```

Comment:
Minor nit: maybe use the name `embeddingConfig` instead of `embeddingFunction` since there can be multiple functions?

Actually, does this support multiple embedding functions?

Reply:
the current implementation does not, but it should be pretty easy to add that in a follow up.